bin/install/install-optout-dcos-cli.sh

```diff
@@ -152,7 +152,7 @@ if [ -z "$DCOS_CLI_VERSION" ]; then
     if $(compare_version 1.6.1); then
         "$VIRTUAL_ENV_PATH/bin/pip" install --quiet "dcoscli<0.4.0"
     else
-        "$VIRTUAL_ENV_PATH/bin/pip" install --quiet "dcoscli"
+        "$VIRTUAL_ENV_PATH/bin/pip" install --quiet "dcoscli==0.4.12"
```

Comment:
Maybe we can make the `0.5.0` release the release that drops `p27` support. If we do this we can change this to `dcoscli<0.5`.